tck/faces-tck/src/com/sun/ts/tests/jsf/common/servlets/HttpTCKServlet.java

```diff
@@ -219,6 +219,11 @@ private void initFaces(ServletContext context, ServletRequest request,
       // Set up references to the application and facesContext objects
       application = facesContext.getApplication();
       facesContext.setViewRoot(createViewRoot());
+
+      // Set up RenderKit
+      renderKitId = facesContext.getApplication().getViewHandler().calculateRenderKitId(facesContext);
```

Comment:
`String renderKitId ` to fix the compilation issues. Trying to build and run this now.

Reply:
*facepalm*
Thanks.